truffle-config.js

```diff
@@ -38,6 +38,11 @@ module.exports = {
    */
 
   networks: {
+    ganache: {
```

Comment:
Hmm why did not you make it `development`?

Reply:
because then the CI does not work as truffle does not spawn its own ganache instance

Reply:
Yeah I suspected as that is where I left the #70 😅

Hmm damn I would like more to have it the otherway around, but Truffle seems to be very stubborn on using its values without any way of configuration it 🙄 Ok lets use your approach then